Makefile.am

```diff
@@ -145,7 +145,7 @@ endif
 
 # Extra stuff to include in the distribution.
 
-mk_FILES =	Basic.mk mk/msdosdjgpp.mk mk/Amiga.mk mk/Windows32.mk
```

Comment:
@rocky got the mk/msdosdjpp.mk have anything to do with compiling for DOS?

Reply:
Back in ancient days of yore possibly before you were born (circa 1989) there was this marvelous extension to DOS to make it work like a real 32-bit system and allow it to run GCC and thus compile GNU programs. See https://en.wikipedia.org/wiki/DJGPP

People who still have those systems and love it, see https://hackaday.com/2018/05/14/msdos-development-with-gcc/, should use GNU Make. 

If for some reason there is a need for that there, the code could be added back in, but it would _have_ to be done in a way that doesn't numerous files strewn about, and `#ifdefs` all over the place. A better way to do this is to create a single directory with that cap in it, and duplicate the functions that need to be modified. `#ifdef` is fine for small changes, but what we see here in `ar.c`, `arscan.c`, and `function.c` is just a bit over the top. 

In one place we can't even get the indentation of some code right because in one variant it is part of an `else` while in the buld of the cases it isn't.